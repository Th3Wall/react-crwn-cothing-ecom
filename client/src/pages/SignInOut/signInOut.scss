.sign-in-out {
    width: 75vw;
    display: flex;
    justify-content: space-around;
    margin: 30px auto;
}

@media screen and (max-width: 1024px) {
    .sign-in-out {
        flex-direction: column;
        align-items: center;
        width: 90%;
    }
}