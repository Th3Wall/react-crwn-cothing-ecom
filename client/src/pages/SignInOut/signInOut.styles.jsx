import styled from "styled-components";
import { motion } from 'framer-motion';

export const SignInOutWrapper = styled(motion.div)`
    width: 75vw;
    display: flex;
    justify-content: space-around;
    margin: 30px auto;

    @media screen and (max-width: 1024px) {
        flex-direction: column;
        align-items: center;
        width: 90%;
    }
`