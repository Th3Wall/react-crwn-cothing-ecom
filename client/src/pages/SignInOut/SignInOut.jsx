import SignIn from '../../components/SignIn/SignIn';
import SignUp from '../../components/SignUp/SignUp';
import { SignInOutWrapper } from './signInOut.styles';

const SignInOut = () => {
    return (
        <SignInOutWrapper
            initial={{ opacity: 0, y: '50px' }}
            animate={{ opacity: 1, y: 0 }}
            exit={{ opacity: 0, y: '50px' }}
            transition={{ ease: "easeOut", duration: 1 }}
        >
            <SignIn />
            <SignUp />
        </SignInOutWrapper>
    )
}

export default SignInOut;
