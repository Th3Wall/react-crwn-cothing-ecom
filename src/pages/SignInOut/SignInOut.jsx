import SignIn from '../../components/SignIn/SignIn';
import './signInOut.scss';

const SignInOut = () => {
    return (
        <div className="sign-in-out">
            <SignIn />
        </div>
    )
}

export default SignInOut;
