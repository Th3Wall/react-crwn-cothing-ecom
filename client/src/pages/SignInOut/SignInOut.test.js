import React from 'react';
import { shallow } from 'enzyme';
import SignInOut from './SignInOut';

describe('SignInOut', () => {
    it('should display SignInOut component', () => {
        const wrapper = shallow(<SignInOut/>);
        expect(wrapper).toMatchSnapshot();
    })
})