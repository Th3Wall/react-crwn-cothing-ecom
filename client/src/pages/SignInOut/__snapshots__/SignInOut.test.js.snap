// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInOut should display SignInOut component 1`] = `
<Styled(Component)
  animate={
    Object {
      "opacity": 1,
      "y": 0,
    }
  }
  exit={
    Object {
      "opacity": 0,
      "y": "50px",
    }
  }
  initial={
    Object {
      "opacity": 0,
      "y": "50px",
    }
  }
  transition={
    Object {
      "duration": 1,
      "ease": "easeOut",
    }
  }
>
  <Connect(SignIn) />
  <Connect(SignUp) />
</Styled(Component)>
`;
